README.md

```diff
@@ -54,19 +54,19 @@ you must add the following dependencies:
 composer require symfony/dependency-injection symfony/http-kernel
 ```
 
-Enable the bundle on the `AppKernel` class:
+Register the bundle in the kernel of your application:
```

Comment:
Not sure if it worth to remove `AppKernel`. It's the same for 99% of case so it will be simpler for beginners.